.github/workflows/ci.yml

```
@@ -13,7 +13,7 @@ jobs:
   # CI matrix - runs the job in lagom-template.yml with different configurations.
   Lagom:
     if: github.repository == 'LadybirdBrowser/ladybird'
-    name: ${{ matrix.os_name }}, ${{ matrix.arch }}, ${{ matrix.build_preset }}, ${{ matrix.toolchain }}
+    name: ${{ matrix.os_name }}, ${{ matrix.arch }}, ${{ matrix.build_preset }}, ${{ matrix.toolchain }}${{ matrix.name_suffix }}
```

actionlint (1.7.12)

[error] property "name_suffix" is not defined in object type {arch: string; build_preset: string; clang_plugins: bool; container: any; os_name: string; runner_labels: string; toolchain: string}

(expression)